### PR TITLE
Add Search Swift Packages extension

### PR DESCRIPTION
**Pull request by maximkrouk**
## Description

The command searches for swift packages using swiftpackageindex

## Screencast

<p align=center>
image
image
image
</p>

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/swift-package-index/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "author": "maximkrouk",
+  "owner": "capturecontext",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search Swift Packages",
```
Comment (danielsequeira) [inline extensions/swift-package-index/package.json:16]:
```suggestion
      "title": "Search Packages",
```

File: extensions/swift-package-index/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      "subtitle": "SPI",
```
Comment (danielsequeira) [inline extensions/swift-package-index/package.json:17]:
```suggestion
      "subtitle": "Swift Package Index",
```

File: extensions/swift-package-index/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# Swift Package Index Changelog
+
+## [Initial Version] - 2022-07-19
```
Comment (danielsequeira) [inline extensions/swift-package-index/CHANGELOG.md:3]:
```suggestion
## [Initial Version] - 2022-07-26```

Comment (danielsequeira) [inline extensions/swift-package-index/CHANGELOG.md:3]:
```suggestion
## [Initial Version] - 2022-07-26
```

File: extensions/swift-package-index/src/index.tsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            />
+          </ActionPanel.Section>
+          <ActionPanel.Section title="GitHub">
+            <Action.OpenInBrowser
+              title="Open GitHub Repo"
+              shortcut={{ modifiers: ["opt"], key: "enter" }}
+              url={manifest.githubURL}
+            />
+            <Action.OpenInBrowser
+              title="Open Author Github"
```
Comment (danielsequeira) [inline extensions/swift-package-index/src/index.tsx:65]:
```suggestion
              title="Open Author Github Page"
```

**Review by danielsequeira — COMMENTED** [review]
Hi @maximkrouk,

Thank you so much for your contribution.

Just a few nitpicking comments on my side, for the sake of consistency.

File: extensions/swift-package-index/package.json (hunk 1 above)
Comment (maximkrouk) [reply extensions/swift-package-index/package.json:16]: I guess that putting the target into the title should help while searching extensions if you use extensions for both npm and spm for example, even if your suggestion looks a bit neater 🤔

I'm okay with the change, just wondering what you think. Maybe there is a way to add keywords? Don't remember it from the docs...

Comment (danielsequeira) [reply extensions/swift-package-index/package.json]: Yes, you can also use the `keywords` property, we mention it in our docs: https://developers.raycast.com/information/manifest#command-properties

My proposed changed is to have this extension coherent with the rest of our extensions and commands. If you keep the command subtitle as the name of the extension, no need to repeat Swift in the command title.

File: extensions/swift-package-index/CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# Swift Package Index Changelog
+
+## [Initial Version] - 2022-07-26
```
Comment (danielsequeira) [inline extensions/swift-package-index/CHANGELOG.md:3]:
```suggestion
## [Initial Version] - 2022-07-27
```

File: extensions/swift-package-index/package.json (hunk 1 above)
Comment (maximkrouk) [reply extensions/swift-package-index/package.json]: Am I right, that "tokens" from any title are used as keywords anyway and there is no need to manually add some?

File: extensions/swift-package-index/package.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "swift-package-index",
+  "title": "Swift Package Index",
+  "description": "Search for Swift Packages via https://swiftpackageindex.com",
+  "icon": "command-icon.png",
+  "author": "maximkrouk",
+  "owner": "capturecontext",
```
Comment (danielsequeira) [inline extensions/swift-package-index/package.json:8]: @maximkrouk please remove the owner property, we don't add it for public extensions

File: extensions/swift-package-index/package.json (hunk 1 above)
Comment (danielsequeira) [reply extensions/swift-package-index/package.json]: Yes, that is true as well

**Pull request merged**